purchase_requisition_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2016-17 Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Operating Unit in Purchase Requisitions",
+    "version": "10.0.1.0.0",
+    "author": "Eficent Business and IT Consulting Services S.L., "
+              "Serpent Consulting Services Pvt. Ltd.,"
+              "Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "website": "http://www.eficent.com",
+    "category": "Purchase Management",
+    "depends": ["purchase_requisition",
```

Comment:
@aheficent Waiting for [#368](https://github.com/OCA/purchase-workflow/pull/368) to get merged.

I guess then we need to add `purchase-workflow` to the oca_dependencies, right?

Don't know though why the travis is green now?!